frontend/lib/util/clang-integration.cpp

```diff
@@ -143,15 +129,17 @@ const std::vector<std::string>& getCC1Arguments(Context* context,
   auto diags = new clang::DiagnosticsEngine(diagID, &*diagOptions, diagClient);
 
   // takes ownership of all of the above
-  clang::driver::Driver D(clangExe, triple, *diags);
+  clang::driver::Driver D(argsCstrs[0], triple, *diags);
 
   std::unique_ptr<clang::driver::Compilation> C(D.BuildCompilation(argsCstrs));
 
   clang::driver::Command* job = nullptr;
 
   if (usingGpuLocaleModel(context) == false) {
-    // Not a CPU+GPU compilation, so just use first job.
-    job = &*C->getJobs().begin();
+    if (C->getJobs().begin() != C->getJobs().end()) {
```

Comment:
nitpick, but it looks like this is just to check there are jobs -- it would be more readable to just directly use `!C->getJobs().empty()` ([docs](https://clang.llvm.org/doxygen/classclang_1_1driver_1_1JobList.html))